cmd/controller-manager/controller-manager.go

```diff
@@ -19,6 +19,13 @@ import (
 	"k8s.io/component-base/logs"
 
 	"github.com/karmada-io/karmada/cmd/controller-manager/app"
+	_ "github.com/karmada-io/karmada/pkg/controllers/binding"
+	_ "github.com/karmada-io/karmada/pkg/controllers/cluster"
+	_ "github.com/karmada-io/karmada/pkg/controllers/execution"
+	_ "github.com/karmada-io/karmada/pkg/controllers/hpa"
+	_ "github.com/karmada-io/karmada/pkg/controllers/mcs"
+	_ "github.com/karmada-io/karmada/pkg/controllers/namespace"
+	_ "github.com/karmada-io/karmada/pkg/controllers/status"
```

Comment:
This is why I don't like it. People have to import the new controllers explicitly.
I think the previous version is pretty good.

Reply:
I think so if add new controller added, People have to import the new controllers explicitly. I will change it to previous veresion